README.md

```diff
@@ -53,4 +55,6 @@ Above all, the most important thing in accessibility is empathy. In the tech ind
 
 The [Supporting Resources document](supporting-resources.md) in this repo has a lot of great links to help you build a deeper understanding of topics around accessibility. We have a set of guidelines and suggested tools for developers in the [Product and Engineering document](product-and-engineering.md). You can also take a look at our resources and activities for [Global Accessibility Awareness Day](global-accessibility-awareness-day).
 
-To help build the skills you need to develop accessible websites, we recommend the [Web Accessibility course from Udacity](https://www.udacity.com/course/web-accessibility--ud891). You’ll need a good grasp of HTML, CSS and vanilla Javascript, but it doesn’t require you to use React or any other special libraries or tools.
+To help build the skills you need to develop accessible websites, we recommend the [Web Accessibility course from Udacity](https://www.udacity.com/course/web-accessibility--ud891). You’ll need a good grasp of HTML, CSS and vanilla Javascript, but it doesn’t require you to use React or any other special libraries or tools. 
+
+You might also like to try this [course from the W3C on EdX (currently free to audit)](https://www.edx.org/course/web-accessibility-introduction). It's based on the W3C's [open curricula](https://www.w3.org/WAI/curricula/) and is aimed at developers, designers, project managers and more.
```

Comment:
Do you think we should just recommend one course i.e. remove the link to the Udacity course? We wouldn't want people to think they need to take lots of accessibility courses, it might feel overwhelming.